packages/office-ui-fabric-react/src/components/Calendar/Calendar.test.tsx

```diff
@@ -332,4 +332,47 @@ describe('Calendar', () => {
     expect(prevMonth.classList.contains('ms-DatePicker-prevYear--disabled')).toBe(true);
     expect(nextMonth.classList.contains('ms-DatePicker-nextYear--disabled')).toBe(true);
   });
+
+  describe('Test Rendering Calendar with Year Picker', () => {
+    let renderedComponent: Calendar;
+    let defaultDate: Date;
+    beforeAll(() => {
+      defaultDate = new Date(2017, 2, 16);
+      renderedComponent = ReactTestUtils.renderIntoDocument(
+        <Calendar strings={dayPickerStrings} isMonthPickerVisible={true} value={defaultDate} />
+      ) as Calendar;
+    });
+    it('month header should have button role', () => {
```

Comment:
nit: it would be great to have a new line in between each of these tests, thanks for the tests though!

Reply:
👍